test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -551,5 +551,22 @@ public async Task AddProductDefaultPKAndDifferentColumnOrderTest(SupportedLangua
             await this.SendOutputGetRequest("addproductdefaultpkanddifferentcolumnorder", null, TestUtils.GetPort(lang, true));
             Assert.Equal(1, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithDefaultPK"));
         }
+
+        /// <summary>
+        /// Tests that when using a table with column names having slash (/ or \) in them 
+        /// we can insert data using output binding.
+        /// Excluding C#, JAVA and CSX languages since / or \ are reserved characters and are discouraged to use in variable names
+        /// </summary>
+        [Theory]
+        [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.CSharp, SupportedLanguages.OutOfProc, SupportedLanguages.Java, SupportedLanguages.Csx)]
+        public async Task AddProductWithSlashInColumnName(SupportedLanguages lang)
+        {
+            Assert.Equal(0, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithSlashInColumnNames"));
+            this.StartFunctionHost("AddProductWithSlashInColumnName", lang);
+            await this.SendOutputPostRequest("addproduct-slashcolumns", "");
+            // Check that a product should have been inserted
+            Assert.Equal(1, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithSlashInColumnNames"));
```

Comment:
Shouldn't this test be verifying that the value in the columns is valid? If I'm reading the issue correctly - it inserted a row but the values in the columns with slashes were just NULL